faucet.config.js

```diff
@@ -3,7 +3,7 @@ let path = require("path");
 
 module.exports = {
 	styleguide: [{
-		source: "./pages.json",
+		source: "./pages.js",
```

Comment:
Different idea:

```js
source: require("./pages")
```

This way, we can reserve the String for the Roman-style configuration where you just provide a path and we will detect all md Files for you and deduce the page structure from there. So the contract is:
* Either provide a path, then we will do it Roman-style
* Or provide a data structure, then we will do it FND-style
With the second option, you can either a) put it in the faucet.config.js directly b) put it in a JS file and require it or c) put it in a JSON file and require it

Reply:
I like it!

Hover, I'm currently deep in the weeds [preparing the next PR](https://github.com/moonglum/styleguide/compare/master...FND:uris), which directly affects this processing:
https://github.com/FND/styleguide/blob/decf0a6a22bf5cbaa8eee033121893485ee83121/lib/site.js#L56
Would you mind putting off this change until that part's stable? It seems easier to adjust over there and would save me the trouble of rebasing with conflicts.